bridge/_files.php

```diff
@@ -20,6 +20,7 @@
     __DIR__ . '/../src/DDTrace/StartSpanOptionsFactory.php',
     __DIR__ . '/../src/DDTrace/Time.php',
     __DIR__ . '/../src/DDTrace/Transport/Http.php',
+    __DIR__ . '/../src/DDTrace/Transport/Internal.php',
```

Comment:
Is this a left-over or some magic happens here? Because I cannot find the file.
In case that case 1 holds true, we should probably have a check here to avoid that files that do not exist can be added to `_files.php`.

Reply:
^^ not as part of this PR of course

Reply:
It's indeed a leftover

Reply:
The file does exist, but it is just not being used, correct?